packages/core/test/lib/commands/compile.js

```diff
@@ -57,6 +57,7 @@ describe("compile", function () {
         quiet: true
       })
     );
+    console.log("length", Object.keys(contracts).length);
```

Comment:
Should this be a `debug`?  Did you mean to delete this before PRing?  Or is this deliberate?

Reply:
Yea, I was going to make a draft PR to rebase before marking it ready. But since it picked up review, I'll be more careful in the future. Though I'll likely do a rebase at some point to drop some unrelated commits

Reply:
Removed.